ci/azure-pipelines.yml

```diff
@@ -57,6 +57,8 @@ jobs:
         displayName: Install SoftHSM
       - script: make unit-test
         displayName: Run Unit Tests
+      - script: make profile
+        displayName: Code Coverage
```

Comment:
Isn't this running unit tests again with profile mode on? Why would we do it twice?

Reply:
Also, this is running as part of the PR. Do we really need code coverage in PR checks?  Maybe we do but I would value fast execution over a report.